boards/arm/mimxrt1060_evk/doc/index.rst

```diff
@@ -333,6 +333,12 @@ etc.):
 - Parity: None
 - Stop bits: 1
 
+Using SWO
+---------------------------------
+SWO can be used as a logging backend, by setting ``CONFIG_LOG_BACKEND_SWO=y``.
```

Comment:
Did you render this HTML and make sure it looks good? It doesn't seem right to me. I would have expected:

```
Using SWO
---------

SWO can be used [...]
```

Reply:
Fixed now, verified the HTML looks good